plugins/cachekey/pattern.cc

```diff
@@ -452,6 +452,18 @@ MultiPattern::name() const
   return _name;
 }
 
+bool
+MultiPattern::capture(const String subject, StringVector &result) const
```

Comment:
can you use `const String&` or `StringView`?

Reply:
Good catch, it is meant to be `const String&`